lib/active_storage_validations.rb

```diff
@@ -20,8 +20,8 @@ def validate_each(record, attribute, value)
       files = Array.wrap(files)
 
       files.each do |file|
-        unless content_type_valid?(file) 
-          record.errors.add(attribute, options[:message].presence || :invalid)
+        unless content_type_valid?(file)
+          record.errors.add(attribute, options[:message].presence || "does not have an authorized content type, authorized content types : #{types}")
```

Comment:
@tleneveu  this is something that we can discuss. I think ideally gem should be without hard-coded language. Maybe with a new key and ability to localize it. What do you think? This is why I put :invalid, since I think with different format it would be translated automatically

Reply:
also if only one type is specified it should output string like "authorized content type : #{types}" (without "s")
and maybe not showed as array, but in more human readable format (maybe joined with ",")

Reply:
Effectively

I think it would be better to add internationalization for providing multi-language support